test/io/input/pipeprocesstester.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+sum_to () (
+    set -- $(seq $1)
+    IFS=+
```

Comment:
I actually have no idea what this is doing, I got it online. Bash is just awful.